pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Enterprise apps on Amazon EKS. In about a day.
+excerpt: Build battle-tested Amazon EKS clusters, designed for enterprise with security, observability, and usability built-in. Manage 100% of your infrastructure using code, all in ~1 day. <b>Join now for special early access pricing!</b>
+permalink: /eks-for-enterprise/
+custom_js:
+- prism
+prism_css: /assets/css/prism-landing-zone.css
+
+technologies:
+- name: Spring Boot
+  logo: /assets/img/logos-technologies/logo-spring-boot.png
+- name: Ruby on Rails
+  logo: /assets/img/logos-technologies/logo-rails.png
+
+hero:
+  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  cta: Request early access
+  logos:
+  - /assets/img/logos-technologies/partner_logo_aws.png
+  - /assets/img/logos-technologies/partner_logo_kubernetes.png
+  - /assets/img/logos-technologies/partner_logo_terraform.png
+
+how_it_works:
+  - heading: "Step 1: Choose your EKS configuration"
+    precode: |
+      <figure><figcaption>eks-configuration.yml</figcaption>
+        <pre>
+            <code class="language-yml">
+            # Configure CIS Benchmark compliance
+            cis:
+              enabled: true
+
+            # Choose from istio, consul-connect, aws-app-mesh
+            service_mesh: "istio"
+
+            # Choose from flux, jenkins, argocd
+            cicd: "flux"
+
+            # Choose from prometheus, datadog, cloudwatch
+            observability: prometheus
+
+            # Configure various common utilities
+            cluster_autoscaler:
+              enabled: true
+
+            sealed_secrets:
+              enabled: true
+
+            aws_load_balancer_controller:
+              enabled: true
+
+            external_dns:
+              enabled: true
+              domain: gruntwork.io
+            </code>
+        </pre>
+      </figure>
+    image_side: right
+    body: |
+      Pick and choose the technologies that you want included in your custom EKS cluster. We
+      support a wide range of popular configurations that include:
+      <ul>
+          <li>Service meshes (Istio, Consul connect, AWS App Mesh)</li>
+          <li>GitOps CI/CD (Flux, Jenkins, Argo)</li>
+          <li>UI dashboards (Porter, Grafana)</li>
+          <li>Observability (Prometheus, Datadog, CloudWatch, ELK)</li>
+          <li>And more (cluster-autoscaler, sealed secrets, AWS Load Balancer Controller, etc)</li>
+      </ul>
+      On top of that, all our enterprise-grade EKS clusters are hardened according to the <a
+      href="https://aws.amazon.com/blogs/containers/introducing-cis-amazon-eks-benchmark/">CIS Amazon EKS
+      Benchmark</a>, regardless of the chosen technologies.
+    logos:
+      - title: CIS
+        src: /assets/img/logos-compliance/cis-benchmark.png
+      - title: EKS
+        src: /assets/img/logos-technologies/logo-eks.png
```

Comment:
Love these logos! Consider adding the logos for:

- Istio
- Flux

Reply:
This is now done in the latest verison.